### ISSUE TITLE
Value updates IWatch 

### ISSUE THREAD

**Issue by yvetvisser** [nightscout/nightguard#181]
I’m not getting value updates on the Apple Watch. I only get updates when I tap on the value and open the app. Does anyone have a solution for this problem?

**Comment by dhermanns** [comment]
I think you ment the nightguard complications on the watch, right?

This can have various reasons I wrote them down here:

https://h3rmanns.medium.com/enabling-background-updates-for-you-iphone-and-watch-4b8167a70b24

**Comment by yvetvisser** [comment]
Yes, I meant the complications on the watch. Thank you for the link! I checked the settings and they are all activated. Not solved yet…

**Comment by dhermanns** [comment]
We had a view more users here that also had problems with background updates. But, beside what I've documented, absolutely no idea what else could be wrong with your setup.

You can have a look at the last tab of the nightguard watch app. It prints out whether your watch was able to do background updates and how often this happened.

**Comment by yvetvisser** [comment]
Well, it says 4.19 updates an hour. Maybe that’s the problem. I would like to have an update every 5 minutes, just like the dexcom app

**Comment by dhermanns** [comment]
Your original posts noted you only get updates by tapping. So you get 4 background updates an hour. Everyone would like to have more updates, but apple is very restrictive regarding these updates.

Is the dexcom app connected to your sensor and communicates using bluetouth?

**Comment by yvetvisser** [comment]
I’m the follower of my daughter. Her dexcom app is connected to her sensor. We use bluetooth

**Comment by dhermanns** [comment]
And your Dexcom follower App updates every 5 minutes on your (follower)
Apple Watch? I‘m not using it - so just curious how they are doing it. I
would expect that you have to have a Dexcom account on your follower iPhone.

Then they could send silent push notifications to do the background
updates. Since Nightscout is not centrally hosted, this isn’t possible.

yvetvisser ***@***.***> schrieb am Fr. 4. Feb. 2022 um 23:29:

**Comment by yvetvisser** [comment]
My daughter has the Dexcom app on her phone. Her sensor is sending her glucoselevels every 5 minutes. I have Dexcom follow on my phone, but I can't see her values on my watch. I only get alarms when she's to high or low. That's why I've installed nightguard. I used to use Sugarmate. This was great! But now they have problems outside the USA. 
Do you think that Dexcom follow can send silent push notifications which I can get on my watch?

**Comment by dhermanns** [comment]
I don’t know the Dexcom App. I thought you wanted to tell me that 5 minutes
updates are working on that app.

Apple is quite mysterious about how you can obtain how much background
updates. And the best is to send silent pushbacks. But, as written before,
this is not possible due to the Nightscout architecture.

But if you get alarms as push, see all values every 15 minutes or actual
values by tapping, that would be ok for me.

yvetvisser ***@***.***> schrieb am Sa. 5. Feb. 2022 um 08:58:

**Comment by dhermanns** [comment]
Let me know if you have more thoughts about this issue...

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FV31TP05EQC6B1C591CWTANS&k=47d0611cc084cfa22dc88eb0c1cfb360&m=WUxwcT&r=RaxVSmu).
TASTERMONIAL
